lib/matplotlib/ticker.py

```diff
@@ -1268,7 +1268,9 @@ def format_eng(self, num):
         # Taking care of the cases like 999.9..., which may be rounded to 1000
         # instead of 1 k.  Beware of the corner case of values that are beyond
         # the range of SI prefixes (i.e. > 'Y').
-        if float(format(mant, fmt)) >= 1000 and pow10 < max(self.ENG_PREFIXES):
+        _fmant = float(format(mant, fmt))
```

Comment:
Is there a particular reason to mark the variable as private? It's local anyway. Also I'd use a more telling name like `rounded_mant`.

Reply:
I followed @jklymak and changed the explicit positive and negative check to a absolute value check. This eliminates the variable completely.